Library/Homebrew/test/support/lib/startup/config.rb

```diff
@@ -15,6 +16,13 @@
   ENV[k] = dir
 end.freeze
 
+# Use a shorter HOMEBREW_TEMP path so Sequoia doesn't error out as often on long paths (> 104 bytes).
+# Use the minimal amount of randomness to avoid collisions while allowing parallel tests.
+require "securerandom"
+random_hex = SecureRandom.hex(1)
+HOMEBREW_TEMP = Pathname("#{homebrew_temp}/brewtests#{random_hex}".squeeze("/")).freeze
```

Comment:
Does this get removed? Do we need an `at_exit` here too?

We previously had `TEST_TMPDIR` `at_exit` to do this but now that this is no longer inside that it sounds like it will stick around?

Reply:
> Does this get removed?

Not beyond what we already do.

> We previously had `TEST_TMPDIR` `at_exit` to do this but now that this is no longer inside that it sounds like it will stick around?

My experience when testing this with `main` was that we're basically never cleaning up these files 😭. I agree: it'd be nicer to clean them up.